internal/pkg/util/goversion/goversion.go

```diff
@@ -7,7 +7,12 @@
 
 package goversion
 
-// Check checks if we are running with the minimal recommended version of Go runtime
-func Check() error {
-	return nil
-}
+// __BUILD_REQUIRES_GO_VERSION_1_10_OR_LATER__ provides a human-readable
+// error message when building this package with an unsupported version
+// of the Go compiler.
+//
+// Keep the name of this variable in sync with the minimum required
+// version specified in the build tag above.
+//
+// nolint:golint
```

Comment:
and this is the reason for wanting to switch to golangci-lint: to be able to add exceptions :-\

The ALL CAPS variable is fine in this case, as it works better for the intended purpose in that way.

Reply:
btw as I know such comments are not indented with space in order not to include them in docs, so better do `//nolint:golint` instead of `// nolint:golint`